.tool-versions

```diff
@@ -1,2 +1,3 @@
 python 3.11.8
 poetry 1.5.0
+jq 1.6
```

Comment:
Could you add a readme for the new script for how to use it and what it does please